frontend/src/components/Gym/GymHistory.tsx

```diff
@@ -69,7 +69,7 @@ const GymHistory = () => {
           <div key={entry.date}>
             <Date>{unixToFullDate(entry.date)}</Date>
             {entry.details
```

Comment:
would it be better to order these before storing it in the redux store? (ie do on actions after fetching)

Reply:
o yea this makes more sense, will change this